### DISCUSSION TITLE
I would like to automatically add borders or shadows to the images.

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#129] [Feature Requests]
### What's your feature/idea?

I would like GitBook to add support for shadows/borders for images. I am concerned that the images I upload are hard to differentiate from the text. 

### What's your desired outcome?

I want my white images to be easily distinguished from the text. I can manage this by managing my screenshots externally but having this feature in GitBook would reduce my workload. 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by ccontinisio** [comment]
My thoughts on this:

In the new redesign of 2 months ago, all images and all drawings now have a border. This goes against me sometimes, as I'd like to have images with transparency, or perhaps blend into the background. But with a forced border, I can never achieve the effect.

This is also very annoying with drawings (the ones created directly in Gitbook).
- I have a drawing of a diagram, and it looks great in bright mode.
- However, it's unreadable in dark mode (especially the arrows).
- So I add a bright background to the drawing, a bit rectangle, so that the arrows are readable in both bright and dark mode.
- When I save the drawing, Gitbook adds a padding to it, and a border. So now the drawing has: a bright background, then a transparent padding, and then a border.

You can see how the forced border doesn't work for many cases. So, could it be optional?
Perhaps on by default, but configurable per-image?

Thanks.

**Comment by meshalA1** [comment]
عن اي رسومات تتحدث من انت
